sql/patch.sql

```diff
@@ -147,3 +147,106 @@ KEY (`success`)
+#EndIf
+
+SET @ai_exist =(SELECT COUNT(*) FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = 'document_templates' AND COLUMN_NAME = 'id' AND 'EXTRA' LIKE '%auto_increment%' LIMIT 1);
+SET @query = IF (@ai_exist = 0, 'ALTER TABLE `document_templates` CHANGE `id` `id` BIGINT(21) UNSIGNED NOT NULL AUTO_INCREMENT', '');
+PREPARE statement FROM @query;
+EXECUTE statement;
```

Comment:
can't this change just be made directly in the above document_templates table definition?

Reply:
yes that'd make sense however, I have to track all the pre testing patches I have in the wild so treat this as upgrade...